mail-filter/amavisd-new/amavisd-new-2.12.0-r1.ebuild

```diff
@@ -0,0 +1,198 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit systemd
+
+DESCRIPTION="High-performance interface between the MTA and content checkers"
+HOMEPAGE="https://gitlab.com/amavis/amavis"
+SRC_URI="https://gitlab.com/amavis/amavis/-/archive/v${PV}/amavis-v${PV}.tar.bz2"
+
+LICENSE="GPL-2 BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~hppa ~ppc ~ppc64 ~sparc ~x86"
+IUSE="clamav courier dkim ldap mysql postgres qmail razor rspamd rspamd-https snmp spamassassin zmq"
+
+MY_RSPAMD_DEPEND="( dev-perl/JSON dev-perl/HTTP-Message dev-perl/LWP-UserAgent-Determined )"
+BDEPEND="acct-user/amavis"
+RDEPEND="${BDEPEND}
+	>=dev-lang/perl-5.10.0
+	app-arch/arc
+	app-arch/bzip2
+	app-arch/cabextract
+	app-arch/cpio
+	app-arch/gzip
+	app-arch/lha
+	app-arch/lrzip
+	app-arch/lzop
+	app-arch/ncompress
+	app-arch/p7zip
+	app-arch/pax
+	app-arch/unarj
+	app-arch/unrar
+	app-arch/xz-utils
+	app-arch/zoo
+	clamav? ( app-antivirus/clamav )
+	>=dev-perl/Archive-Zip-1.14
+	dev-perl/BerkeleyDB
+	dev-perl/Convert-BinHex
+	dev-perl/File-LibMagic
+	dev-perl/IO-Socket-SSL
+	dev-perl/IO-stringy
+	>=dev-perl/Mail-DKIM-0.31
+	>=dev-perl/MailTools-1.58
+	>=dev-perl/MIME-tools-5.415
+	dev-perl/Net-LibIDN
+	>=dev-perl/Net-Server-0.91
+	dev-perl/Net-SSLeay
+	dev-perl/Unix-Syslog
+	ldap? ( >=dev-perl/perl-ldap-0.33 )
+	mysql? ( dev-perl/DBD-mysql )
+	net-mail/ripole
+	net-mail/tnef
+	postgres? ( dev-perl/DBD-Pg )
+	razor? ( mail-filter/razor )
+	rspamd? ( ${MY_RSPAMD_DEPEND} )
+	rspamd-https? ( ${MY_RSPAMD_DEPEND} dev-perl/LWP-Protocol-https dev-perl/Net-SSLeay )
+	snmp? ( net-analyzer/net-snmp[perl] )
+	spamassassin? ( mail-filter/spamassassin dev-perl/Image-Info )
+	>=sys-apps/coreutils-5.0-r3
+	>=sys-libs/db-4.4.20
```

Comment:
Could help by adding a slot.

Reply:
Sorry, I don't know what you mean.

Reply:
`>=sys-libs/db-4.4.20:*`, doesn't matter much.